### ISSUE TITLE
Speak fx type in effect chooser 

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6491]
When I'm using the FX controls I notice that the currently selected FX radio button is filled.  This is good.  What I'm missing though is any feedback on which slots are already filled.  Or rather, to get this feedback, I need to select a slot such as pressing enter on stage A slot 1, then tabbing lots to get to the FX selector to find out if it is empty or not.  Is there a visual indicator that shows a slot is already filled, and if so, is it possible to get that info via a SR?  Or am I missing an existing method?

From discord user chessel
